A3-channel-tracing.md

```diff
@@ -43,6 +43,19 @@ _Trace events_ should only be added for events that happen relatively infrequent
 ... Entering idle mode
 ```
 
+## Minimal Set of Data
+
+We define a minimal set of events that must be traced, in order to consider the channel tracing feature as complete.
+
+All of these events must be traced:
+- Channel creation and deletion
+- Subchannel creation and deletion
+- Address resolution resulting from service config change
+- Address resolution that causes number of backends to go from zero to non-zero
```

Comment:
The address resolution case is complex enough that it warrants more than just a couple of bullets in this list.  Let's have a separate paragraph explaining why we need to be selective about which address resolution events get recorded, and then have a separate bulleted list identifying the cases in which we should log resolver results.

Note that we also need to record the address resolution result whenever it causes us to create a new LB policy.

Reply:
Done